ovs/actionparser_test.go

```diff
@@ -281,6 +281,18 @@ func Test_parseAction(t *testing.T) {
 			s: "set_field:192.168.1.1->arp_spa",
 			a: SetField("192.168.1.1", "arp_spa"),
 		},
+		{
+			s: "conjunction(123, 1/2)",
+			a: Conjunction(123, 1, 2),
+		},
+		{
+			s: "conjunction(123, 2/2)",
+			a: Conjunction(123, 2, 2),
+		},
+		{
+			s:       "conjunction(123, 3/2)",
+			invalid: true,
+		},
```

Comment:
These tests didn't catch the fact that the code was doing `patConjunction[:len(patModVLANVID)-9]` -- maybe throw one in to cover that & check that it doesn't catch a similar-but-garbage prefix (`conjjjjj`, or something).

Reply:
Totally right - added